src/main/java/uk/gov/hmcts/reform/cosapi/exception/InvalidResourceException.java

```diff
@@ -1,14 +0,0 @@
-package uk.gov.hmcts.reform.cosapi.exception;
-
-public class InvalidResourceException extends RuntimeException {
-
-    private static final long serialVersionUID = 7442994120484411077L;
```

Comment:
Is this Deletion required? @saravanakumar-palanichamy , could you please check on this.

Reply:
Exception was not used at all. Redundant.